src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs

```diff
@@ -608,6 +612,25 @@ where FilterVisitor.CanVisitApi(t)
             }
         }
 
+        private void GenerateExtensionMethods(INamedTypeSymbol symbol, MetadataItem item)
+        {
+            var extensions = new List<string>();
+            foreach (var e in _extensionMethods)
+            {
+                var reduced = e.ReduceExtensionMethod(symbol);
+                if ((object)reduced != null)
+                {
+                    // update reference
+                    var typeGenericParameters = e.ContainingType.IsGenericType ? e.ContainingType.Accept(TypeGenericParameterNameVisitor.Instance) : EmptyListOfString;
+                    var methodGenericParameters = e.IsGenericMethod ? (from p in e.TypeParameters select p.Name).ToList() : EmptyListOfString;
+                    var id = AddSpecReference(e, typeGenericParameters, methodGenericParameters);
```

Comment:
the reference need to be the original api's uid so it could be linked. i will see whether we could get the updated name info.

---
In reply to: [65019055](https://github.com/dotnet/docfx/pull/339#discussion_r65019055) [](ancestors = 65019055)